recipes/libmeshb/all/test_package/test_package.c

```diff
@@ -0,0 +1,29 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include "libmeshb7.h"
```

Comment:
```suggestion
#include <libmeshb7.h>

#include <stdio.h>
#include <stdlib.h>
```
Not critical, but it's useful to list the project include first to test that no standard headers are missing from the project includes.